stmt_test.go

```diff
@@ -194,6 +194,18 @@ func (s *stmtSuite) TestFixDoubleInterpolateInStmt() {
 	s.NoError(rows.Close())
 }
 
+func (s *stmtSuite) TestFixMultiLineInsert() {
```

Comment:
Please add a new testcase for test TestExecMulti. 
like this:
```go
queries = []string{"INSERT INTO data (i64) VALUES (?)", ""INSERT INTO data(\ni64\n) VALUES (\n?\n)"}
for q in queries:
   //.. test case logic
```

Reply:
this test can be removed, since TestExecMulti covers this case

Reply:
done